src/instruments/src/SD/Pages/Crz/common.tsx

```diff
@@ -0,0 +1,27 @@
+export const splitDecimals = (value, type) => {
```

Comment:
This file is called `common`, but what is it common to ?

Reply:
Just common to the cruise page at the moment, but I might use some of these functions again in which case they could be moved to SD > Common. Or I can move it now?